plugins/vortex-mod-youtube/README.md

```
@@ -0,0 +1,44 @@
+```bash
+rustup target add wasm32-wasip1
+cargo build --release
+```
+
+The resulting WASM binary is at `target/wasm32-wasip1/release/vortex_mod_youtube.wasm`.
+
+## Install
+
+Copy the `.wasm` and `plugin.toml` to `~/.config/vortex/plugins/vortex-mod-youtube/`:
+
+```
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)